### PR TITLE
#253 Extract dialog

### PR DESCRIPTION
**Pull request by michal-kucznerowicz**

### CODE REVIEW

**Review by bartek977 — APPROVED** [review]

File: app/src/main/kotlin/com/intive/picover/profile/view/DeleteAccountDialog.kt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		},
+		text = {
+			Text(stringResource(R.string.DeleteAccountConfirmationDialogDescription))
+		},
+	)
+}
+
+@LightDarkPreview
+@Composable
+private fun DeleteAccountDialog() {
```
Comment (MarekMacko) [inline app/src/main/kotlin/com/intive/picover/profile/view/DeleteAccountDialog.kt]: Add suffix `Preview`

File: app/src/main/kotlin/com/intive/picover/profile/view/DeleteAccountDialog.kt [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	PicoverTheme {
```
Comment (MarekMacko) [inline app/src/main/kotlin/com/intive/picover/profile/view/DeleteAccountDialog.kt:41]: Is there a way to set this our theme for all previews?

Comment (michal-kucznerowicz) [reply app/src/main/kotlin/com/intive/picover/profile/view/DeleteAccountDialog.kt:41]: I was looking for it but found nothing

File: app/src/main/kotlin/com/intive/picover/profile/view/DeleteAccountDialog.kt (hunk 1 above)
Comment (michal-kucznerowicz) [reply app/src/main/kotlin/com/intive/picover/profile/view/DeleteAccountDialog.kt]: Done

**Review by MarekMacko — APPROVED** [review]

**Pull request merged**